build.gradle.kts

```diff
@@ -67,6 +67,9 @@ tasks.build {
     dependsOn(tasks.shadowJar)
 }
 
+tasks.register("getVersion") {
+    println(version)
+}
 
 fun execute(vararg command: String): String = ByteArrayOutputStream()
```

Comment:
This can be removed too now

Reply:
See other comment >:(

Reply:
This can't be removed btw because i need to be able to get the version from the script

Reply:
See line 28

Reply:
I mean the `execute` function